mantaray/marshal.go

```diff
@@ -95,13 +100,17 @@ func (n *Node) UnmarshalBinary(bytes []byte) error {
 	offset := preambleSize
 	bb := &bitsForBytes{}
 	bb.fromBytes(bytes[32:])
-	bb.iter(func(b byte) error {
+	err := bb.iter(func(b byte) error {
```

Comment:
Looks to me that the error below is not needed.

```go
return bb.iter(func(b byte) error {
...
```

Reply:
Can be made like that for now. The current code is due to some future changes.